static_src/components/icon.jsx

```diff
@@ -58,8 +49,7 @@ export default class Icon extends React.Component {
     const sizeClass = this.props.iconSize && `icon-${this.props.iconSize}`;
     const borderedClass = (this.props.bordered) &&
       'icon-bordered';
-    const iconClasses = this.styler(mainClass, styleClass, sizeClass,
-                                    borderedClass);
+    const iconClasses = `${mainClass} ${styleClass} ${sizeClass} ${borderedClass}`;
```

Comment:
Should you change this var and the vars above it to use `classNames` while you're here?